src/SQLChecks/Tests/SQLInstance.tests.ps1

```diff
@@ -49,4 +49,10 @@ Describe "Database Mail is configured" -Tag DatabaseMail {
 	It "Database mail is running on $serverInstance" {
 		@(Get-DatabaseMailStatus -Config $Config).Status | Should Be "STARTED"
 	}
+}
+
+Describe "SQL Alerts" -Tag SQLAlerts {
```

Comment:
Could we pick a more descriptive tag - tags are globally unique in the module and drive config rules, so something that is verbose is better - maybe UnconfiguredSQLAgentAlerts ?